src/components/CustomButton/CustomButton.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import {
+    Button,
+    withStyles
+} from '@material-ui/core'
+
+export default withStyles(theme=>({
+
+}))(props=>{
+    const{position,right,noShadow, className, variant, color,backgroundColor, size,children,borderRadius, ...rest}=props
+    return(
+        <Button
+          className={className}
+          variant={variant}
+          size={size}
+          //color={color}
+          style={{
+            color: color==='primary' ?'#000000'
+                :color==='default'?'#686868'
+                :{color},
+            backgroundColor: color==='primary' ?'#FDCC4F'
+                :color==='default'?'#EEEEEE'
+                :{backgroundColor},
```

Comment:
color and backgroundColor should be implemented in color prop `color={color}` and the color value is setting in the `theme.js`.